### PR TITLE
Run format and fix doctests

### PR DESCRIPTION
**Pull request by graingert**

### CODE REVIEW

**Comment by graingert** [comment]
@jklukas @thisfred @bouk: Thoughts?

File: redshift_sqlalchemy/dialect.py [hunk 1]
```diff
@@ -20,7 +20,8 @@ class RedshiftImpl(postgresql.PostgresqlImpl):
 
 
 class RedShiftDDLCompiler(PGDDLCompiler):
```
Comment (bouk) [inline redshift_sqlalchemy/dialect.py]: This should be RedshiftDDLCompiler (uncapitalized S)

Comment (graingert) [inline redshift_sqlalchemy/dialect.py]: That should be a different PR.

File: redshift_sqlalchemy/dialect.py [hunk 2]
```diff
@@ -28,39 +29,87 @@ class RedShiftDDLCompiler(PGDDLCompiler):
...
+    ...     redshift_distkey='id',
+    ...     redshift_sortkey=['id', 'name'],
+    ... )
+    >>> print(CreateTable(user).compile(engine))
+    <BLANKLINE>
+    CREATE TABLE "user" (
+        id INTEGER NOT NULL,
+        name VARCHAR,
+        PRIMARY KEY (id)
+    ) DISTSTYLE KEY DISTKEY (id) SORTKEY (id, name)
```
Comment (jklukas) [inline redshift_sqlalchemy/dialect.py]: I'm of the opinion that these table attributes should be separated by newlines, but that would also be a change for a separate PR.

**Comment by jklukas** [comment]
+1
